### DISCUSSION TITLE
how to change the max upload limit etc on the alpha

### DISCUSSION THREAD

**Discussion by Baltimorepc** [LycheeOrg/Lychee#2557] [Q&A] [answered]
how do i change the limits. in the conf file there is no user.ini. I know use are busy and if its not possible now no worries i just know it was easy to change these before but now i cant find them anywhere 
 Max uploaded file size:                    100M
Max post size:                             100M
Chunk size:                                25.60 MB
Max execution time:                        3600 

just i know there may be alot uploaded at once including high res videos and pictures and i dont want it to fail.

**Comment by ildyria** [comment, marked as answer]
> Max uploaded file size: 100M
> Max post size: 100M

Those are set in your php.ini file, Lychee cannot really modify those.

> Chunk size 

That one you can modify in your settings, look for: _"Size of chunks when uploading in bytes: 0 is auto"_

> just i know there may be alot uploaded at once including high res videos and pictures and i dont want it to fail.

That is the beauty of the version 6. It now supports upload by chunk by default. So when you start uploading, the file is split in your browser in chunks of 25.6 MB, and  repacked on the server side. As a result you are not hit by the limit of upload size (unless you set your chunk size to high number).

**Comment by Baltimorepc** [comment]
Thank you I’ll just modify from there 



Sent from Yahoo Mail for iPhone


On Monday, October 7, 2024, 3:30 AM, Benoît Viguier ***@***.***> wrote:





Max uploaded file size: 100M
Max post size: 100M


Those are set in your php.ini file, Lychee cannot really modify those.


Chunk size


That one you can modify in your settings, look for: "Size of chunks when uploading in bytes: 0 is auto"


just i know there may be alot uploaded at once including high res videos and pictures and i dont want it to fail.


That is the beauty of the version 6. It now supports upload by chunk by default. So when you start uploading, the file is split in your browser in chunks of 25.6 MB, and repacked on the server side. As a result you are not hit by the limit of upload size (unless you set your chunk size to high number).
